rpm/clearwater-socket-factory.spec

```diff
@@ -13,19 +13,26 @@ Enables other processes to establish connections using a different network names
 . %{rootdir}/build-infra/cw-rpm-utils clearwater-socket-factory %{rootdir} %{buildroot}
 setup_buildroot
 install_to_buildroot < %{rootdir}/debian/clearwater-socket-factory.install
-copy_to_buildroot debian/clearwater-socket-factory.service /etc/systemd/system
+copy_to_buildroot debian/clearwater-socket-factory-mgmt.service /etc/systemd/system
+copy_to_buildroot debian/clearwater-socket-factory-sig.service /etc/systemd/system
 copy_to_buildroot debian/clearwater-socket-factory.logrotate /etc/logrotate.d clearwater-socket-factory
+rm -f /etc/init/clearwater-socket-factory-mgmt.conf
+rm -f /etc/init/clearwater-socket-factory-sig.conf
```

Comment:
We'll never have an upstart config file on debian, so these lines are not needed.